tests/test_notifications.py

```diff
@@ -3,9 +3,9 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 """Tests for the notifications API."""
 
+import pytest
```

Comment:
I wonder how these were missed??

Reply:
I think it's because we're now running flake8 with pytest installed, and it's able to correctly identify it as a third party package...

Reply:
Ah that makes sense.